manifests/plugin/dns/infoblox.pp

```diff
@@ -4,19 +4,22 @@
 #
 # === Parameters:
 #
-# $dns_server:: The address of the Infoblox server
+# $dns_server::   The address of the Infoblox server
 #
-# $username::   The username of the Infoblox user
+# $username::     The username of the Infoblox user
 #
-# $password::   The password of the Infoblox user
+# $password::     The password of the Infoblox user
 #
-# $dns_view::   The Infoblox DNS View
+# $dns_view::     The Infoblox DNS View
```

Comment:
I think both the view and DNS timeout can be considered advanced. 

```suggestion
# === Advanced Parameters:
#
# $dns_view::     The Infoblox DNS View
```

Reply:
Updated.